services/core-api/app/api/now_applications/resources/now_application_delay_resource.py

```diff
@@ -79,7 +100,18 @@ def put(self, now_application_guid, now_application_delay_guid):
 
         now_delay = NOWApplicationDelay._schema().load(
             request.json, instance=NOWApplicationDelay.find_by_guid(now_application_delay_guid))
-        now_delay.end_date = datetime.now(tz=timezone.utc)
+
```

Comment:
Not sure if intentional (Or I've missed the function call somewhere) but shouldn't there be validation to check start < end?

Reply:
added!